server/lib/search.js

```diff
@@ -143,6 +157,10 @@ export const searchCollectivesInDB = async (
     dynamicConditions += `AND "countryISO" IN (:countryCodes) `;
   }
 
+  if (tags?.length) {
+    dynamicConditions += `AND "tags" @> (:searchedTags) `;
+  }
+
```

Comment:
With https://github.com/opencollective/opencollective-api/pull/7376 `tags` are now included in the stored TS vector. The search should therefore rely on TS vectors, they're weighted and indexed which makes them both more accurate and performant.

Reply:
Forget about the previous comment. This approach is less performant, but since we're searching for explicitly passed tags (which we know exist in the DB), full-text search while more performant would be less accurate. We don't want to search for these tags in names and long descriptions.

All good to me!